lib/Alchemy/Phrasea/Core/Provider/PhraseaEventServiceProvider.php

```diff
@@ -42,6 +47,27 @@ public function register(Application $app)
         $app['phraseanet.content-negotiation-subscriber'] = $app->share(function (Application $app) {
             return new ContentNegotiationSubscriber($app);
         });
+        $app['phraseanet.record-subscriber'] = $app->share(function (Application $app) {
```

Comment:
I don't think it's a good idea mix and match subscribers of various type together : we already have a service provider for the search engine, since the engine depends on the firing of these event there is little reasons to put subscribers here.